### DISCUSSION TITLE
Offline activation and header content type

### DISCUSSION THREAD

**Discussion by nikolaevich23** [bucanero/pkgi-ps3#78] [Ideas]
1) Please add offline activation? Similar to https://github.com/bucanero/apollo-ps3/tree/master/source
If act.dat not exist - create act.dat 

2) Please add the content type (content_entries) to the title (pkgi_do_head).

**Comment by bucanero** [comment]
hello @nikolaevich23 

I don't have much free time lately for homebrew, so I guess the offline activation will remain on Apollo. Activation in any case is a one-time thing.

Adding the content category at the top should be an easy/simple addition. I'll keep that in mind.

**Reply by LuanTeles** [reply]
> hello @nikolaevich23
> 
> I don't have much free time lately for homebrew, so I guess the offline activation will remain on Apollo. Activation in any case is a one-time thing.
> 
> Adding the content category at the top should be an easy/simple addition. I'll keep that in mind.

I'm interrested on it :P

**Reply by nikolaevich23** [reply]
in my modification i did an [activation check](https://github.com/ErikPshat/pkgi/blob/196557b8f7513a09e3344b9e6e0a539200632e25/source/pkgi_ps3.c#L865) and added some categories. I would add activation, but I ran into the need to connect libraries that I could not find
